### DISCUSSION TITLE
Can you please remove the restriction that it has to have a colon after a word?

### DISCUSSION THREAD

**Discussion by hyperphysin** [marcel-goldammer/obsidian-keyword-highlighter#22] [Ideas]
I wish I could use this plugin to colorize logical symbols such as →, ∴ and ∵ to make grabbing logical structure of my notes easier.

**Comment by marcel-goldammer** [comment]
With [v1.3.0](https://github.com/marcel-goldammer/obsidian-keyword-highlighter/releases/tag/1.3.0), you can now use your keywords without the colon trigger.
